src/PathMaster.ts

```diff
@@ -9,7 +9,7 @@ import { Config } from "./Config";
  * If a import url isn't relative
  * and only has ascii + @ in the name it is considered a node module
  */
-const NODE_MODULE = /^([a-z@].*)$/;
+const NODE_MODULE = /^[^:]([a-z@].*)$/;
```

Comment:
Thought this match was only for `"foo"` or `"foo/bar"` or `"@types/foo"` etc. 

Not full file paths e.g. it doesn't support `/` either. I can be wrong :rose: